#src/index.html

```diff
@@ -18,37 +18,83 @@
                     <ul class="nav__list">
                         <li class="nav__item nav__item--active">
                             <a href="#" class="nav__link">
-
+                                
+                                Home
                             </a>
                         </li>
                         <li class="nav__item">
                             <a href="#" class="nav__link">
                                 
+                                Destinations
                             </a>
                         </li>
                         <li class="nav__item">
                             <a href="#" class="nav__link">
                                 
+                                About
                             </a>
                         </li>
                         <li class="nav__item">
                             <a href="#" class="nav__link">
                                 
+                                Partner
                             </a>
                         </li>
                         <li class="nav__item">
                             <button class="nav__button button button--outline">
-
+                                
+                                Login
                             </button>
                         </li>
                         <li class="nav__item nav__item--btn">
                             <button class="nav__button button button--accent">
                                 
+                                Register
                             </button>
                         </li>
                     </ul>
                 </nav>
             </div>
+            <div class="header__row row">
+                <div class="header__search search col-md-5">
+                    <h1 class="search__block-header block-header">Explore and Travel</h1>
```

Comment:
Is `block-header` used anywhere else? Or why should it be defined as **block**, not just as _element_ of _block_?